### ISSUE TITLE
Error on Startup

### ISSUE THREAD

**Issue by bloggy** [ShaneBeee/SkBee#130]
```
[13:52:00] [Server thread/INFO]: [SkBee] Enabling SkBee v1.14.2
[13:52:00] [Server thread/INFO]: [SkBee] Loading NBTApi...
[13:52:00] [Server thread/INFO]: [SkBee] [NBTAPI] Found Spigot: v1_18_R1! Trying to find NMS support
[13:52:00] [Server thread/INFO]: [SkBee] [NBTAPI] NMS support 'MC1_18_R1' loaded!
[13:52:00] [Server thread/INFO]: [SkBee] [NBTAPI] Found Gson: class com.google.gson.Gson
[13:52:00] [Server thread/INFO]: [SkBee] Successfully loaded NBTApi!
[13:52:00] [Server thread/INFO]: [SkBee] NBT Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Recipe Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Scoreboard Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Bound Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Text Component Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Pathfinding Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] New Structure Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Old Structure Elements DISABLED!
[13:52:00] [Server thread/INFO]: [SkBee]  - Old structure system is no longer available on MC 1.18+
[13:52:00] [Server thread/INFO]: [SkBee]  - Please use new structure system
[13:52:00] [Server thread/INFO]: [SkBee] Other Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Virtual Furnace Elements disabled via config
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[13:52:00] [Server thread/ERROR]: #!#! SkBee's class com.shanebeestudios.skbee.elements.worldcreator.type.Types generated an exception while loading
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[13:52:00] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[13:52:00] [Server thread/ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[13:52:00] [Server thread/ERROR]: #!#! Following plugins are probably related to this error in some way:
[13:52:00] [Server thread/ERROR]: #!#! SkBee (https://github.com/ShaneBeee/SkBee) 
[13:52:00] [Server thread/ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[13:52:00] [Server thread/ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[13:52:00] [Server thread/ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[13:52:00] [Server thread/ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[13:52:00] [Server thread/ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Stack trace:
[13:52:00] [Server thread/ERROR]: #!#! java.lang.IllegalArgumentException: Can't register org.bukkit.World$Environment with the code name environment because that name is already used by environment type
[13:52:00] [Server thread/ERROR]: #!#!     at Skript.jar//ch.njol.skript.registrations.Classes.registerClass(Classes.java:90)
[13:52:00] [Server thread/ERROR]: #!#!     at SK_SkBee.jar//com.shanebeestudios.skbee.elements.worldcreator.type.Types.<clinit>(Types.java:27)
[13:52:00] [Server thread/ERROR]: #!#!     at java.base/java.lang.Class.forName0(Native Method)
[13:52:00] [Server thread/ERROR]: #!#!     at java.base/java.lang.Class.forName(Class.java:467)
[13:52:00] [Server thread/ERROR]: #!#!     at Skript.jar//ch.njol.skript.SkriptAddon.loadClasses(SkriptAddon.java:107)
[13:52:00] [Server thread/ERROR]: #!#!     at SK_SkBee.jar//com.shanebeestudios.skbee.SkBee.loadWorldCreatorElements(SkBee.java:317)
[13:52:00] [Server thread/ERROR]: #!#!     at SK_SkBee.jar//com.shanebeestudios.skbee.SkBee.onEnable(SkBee.java:107)
[13:52:00] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
[13:52:00] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[13:52:00] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
[13:52:00] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:564)
[13:52:00] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:478)
[13:52:00] [Server thread/ERROR]: #!#!     at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:732)
[13:52:00] [Server thread/ERROR]: #!#!     at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:508)
[13:52:00] [Server thread/ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:349)
[13:52:00] [Server thread/ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1218)
[13:52:00] [Server thread/ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:322)
[13:52:00] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Version Information:
[13:52:00] [Server thread/ERROR]: #!#!   Skript: 2.6 (custom version)
[13:52:00] [Server thread/ERROR]: #!#!     Flavor: selfbuilt-unknown
[13:52:00] [Server thread/ERROR]: #!#!     Date: unknown
[13:52:00] [Server thread/ERROR]: #!#!   Bukkit: 1.18.1-R0.1-SNAPSHOT
[13:52:00] [Server thread/ERROR]: #!#!   Minecraft: 1.18.1
[13:52:00] [Server thread/ERROR]: #!#!   Java: 17.0.1 (Java HotSpot(TM) 64-Bit Server VM 17.0.1+12-LTS-39)
[13:52:00] [Server thread/ERROR]: #!#!   OS: Windows 10 amd64 10.0
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Server platform: Paper
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Current node: null
[13:52:00] [Server thread/ERROR]: #!#! Current item: null
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Thread: Server thread
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! Language: english
[13:52:00] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/ERROR]: #!#! End of Error.
[13:52:00] [Server thread/ERROR]: #!#! 
[13:52:00] [Server thread/INFO]: [SkBee] World Creator Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Game Event Elements successfully loaded
[13:52:00] [Server thread/INFO]: [SkBee] Successfully enabled v1.14.2 in 0,20 seconds
```

This server is running Purpur version git-Purpur-1520 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: dec07e6 on HEAD)
SkBee v1.14.2

**Comment by ShaneBeee** [comment]
Looks like another addon already registered "environment", I'll have to add a check to make sure its not already registered.

**Comment by ShaneBeee** [comment]
This has been fixed, could you please download this build
https://github.com/ShaneBeee/SkBee/actions/runs/1741440270
(at the bottom you will see the artifact download, just unzip it and run as usual)

and let me know if it works.

**Comment by ShaneBeee** [comment]
Fixed, hopefully, in 1.15.0
